pilot/pkg/model/jwks_resolver.go

```diff
@@ -42,7 +42,7 @@ const (
 
 	// JwksURI Cache expiration time duration, individual cached JwksURI item will be removed
 	// from cache after its duration expires.
-	jwksURICacheExpiration = time.Hour * 24
+	JwksURICacheExpiration = time.Hour * 24
```

Comment:
Do we still need these contants? And why need to be public?

Reply:
There are a couple of unit tests that are depends on these constants, and the fact that values from flags do not exist when running these tests. Reason why we they need to be public is that they are used outside the package.
https://github.com/istio/istio/pull/11424/files#diff-970c02a18802131086a418f813b9374eR243

Reply:
I mean do they need to be here, or can be defined (private) in the test only.

Reply:
I see. That makes sense. Will make the change.

Reply:
Actually, don't you think it's better to have these constant at one place and all relevant tests (from different packages) can get these values for consistency purposes? There are a few tests that are using these constants.